### PR TITLE
fix(labels): update the WI-label association callout to not confuse with label creation

### PR DESCRIPTION
**Pull request by debloper**
Closes openshiftio/openshift.io#1836

### CODE REVIEW

**Comment by fabric8cd** [comment]
@debloper fabric8/fabric8-ui:SNAPSHOT-PR-2476-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2476-fabric8-planner.badger.fabric8.io

File: src/app/components/label-selector/label-selector.component.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <ng-template #toggleButtonRef>
     <span class="clickable add-label" (click)="closeAddLabel()">
       <i class="pficon pficon-add-circle-o"></i>
-      Add label
+      Assign labels
```
Comment (nimishamukherjee) [inline src/app/components/label-selector/label-selector.component.html]: @Preeticp is it Assign label or Assign Label?

Comment (Preeticp) [reply src/app/components/label-selector/label-selector.component.html]: Ideally, I would suggest Add label, but we have Title case being used in majority of the cases in OSIO, so to keep it consistent let's use Assign Label.

Comment (Preeticp) [reply src/app/components/label-selector/label-selector.component.html]: If it has to be changed it should be across OSIO as a whole to sentence case.

Comment (nimishamukherjee) [reply src/app/components/label-selector/label-selector.component.html]: Thanks @Preeticp 
@debloper please make it "Assign Label"

Comment (debloper) [reply src/app/components/label-selector/label-selector.component.html]: Please reflect whether it should be `Label` or `Labels` cause it deals with multiple association.

Comment (rgarg1) [reply src/app/components/label-selector/label-selector.component.html]: @Preeticp So should it be "Assign Labels" ? Can you confirm.

**Comment by rgarg1** [comment]
Its still "Assign labels" in the snap-shot.

**Comment by debloper** [comment]
> Its still "Assign labels" in the snap-shot.

@rgarg1 please check my comment in the review thread. I haven't gotten a response in favor/against the pluralization from @Preeticp

**Comment by debloper** [comment]
It's better if the confirmation comes from @Preeticp, and as she has amended it to `Assign Label(s)` keeping multiple association in mind (and not only casing, which was the previous suggestion about).

The updated suggestion seems good to me.

**Comment by sanbornsen** [comment]
This change has to be done in the new version. You can find the relative file in **components_ngrx** directory.

**Comment by debloper** [comment]
@sanbornsen I don't see the change reflected. It's still showing `Add label` to me on http://fabric8-ui-fabric8-planner.badger.fabric8.io

screenshot from 2018-03-12 15-06-31

Also, fixing issues that others are already working on without notifying them or on the issues that someone else is taking it over (this is the 4th PR I see your comment, mentioning same thing) is not the best use of time for both the parties. Please notify before hands if you absolutely have to take over an already WIP/submitted/in-review PR.

**Comment by nimishamukherjee** [comment]
@debloper add this change to the ngrx-component.

**Comment by rgarg1** [comment]
@debloper The primary option in the workitem preview is fixed and showing as __Assign Label(s)__ however, there is another _Add Label_ label inside the drop-down that needs to be addressed. See snapshot attached.

add label

**Comment by rgarg1** [comment]
LGTM @debloper

**Pull request merged**